app/models/concerns/user_concern.rb

```diff
@@ -470,15 +470,45 @@ def deactivation_status(user)
       "Account deactivated by #{name} on #{version.created_at}"
     end
 
-    def self.text_search(text)
-      return none unless text.present?
-
-      query = "%#{text}%"
-      where(
-        arel_table[:last_name].matches(query).
-        or(arel_table[:first_name].matches(query)).
-        or(arel_table[:email].matches(query)),
-      )
+    # Search for users by name or email using both prefix matching and fuzzy matching
+    #
+    # @param text [String] the search query
+    # @param sort_by_best_match [Boolean] whether to order results by similarity to query
+    # @param similarity_threshold [Float] minimum similarity score (0.0 to 1.0) for fuzzy matches
+    #
+    # @return [ActiveRecord::Relation] matching users
+    #
+    # @example Find users matching 'john'
+    #   User.text_search('john')
+    #
+    # @example Find users with similarity ordering
+    #   User.text_search('john smith', sort_by_best_match: true)
```

Comment:
You pulled out the fuzzy match and `similarity_threshold` right?
```suggestion
    # Search for users by name or email using both prefix matching
    #
    # @param text [String] the search query
    # @param sort_by_best_match [Boolean] whether to order results by similarity to query
    #
    # @return [ActiveRecord::Relation] matching users
    #
    # @example Find users matching 'john'
    #   User.text_search('john')
    #
    # @example Find users with similarity ordering
    #   User.text_search('john smith', sort_by_best_match: true)
```

Reply:
yes, thanks for catching the stale docs